docs/load-testing.md

```diff
@@ -0,0 +1,73 @@
+# Load testing the OCSP signing components.
+
+Here are instructions on how to realistically load test the OCSP signing
+components of Boulder, exercising the pkcs11key, boulder-ca, and
+ocsp-updater components.
+
+Set up a SoftHSM instance running pkcs11-daemon on some remote host with more
+CPUs than your local machine. Easiest way to do this is to clone the Boulder
+repo, and on the remote machine run:
+
+    remote-machine$ docker-compose run -p 5657:5657 bhsm
+
+Check that the port is open:
+
+    local-machine$ nc -zv remote-machine 5657
+    Connection to remote-machine 5657 port [tcp/*] succeeded!
+
+Set up a handy alias:
+
+    local-machine$ alias drun="docker-compose run -e PKCS11_PROXY_SOCKET=tcp://remote-machine:5657 -e FAKE_DNS=172.17.0.1 --service-ports"
```

Comment:
Is `FAKE_DNS` needed for anything here? It doesn't seem like you are running `dns-test-srv` anywhere when performing the benchmark.